### ISSUE TITLE
Content appears as raw HTML after saving

### ISSUE THREAD

**Issue by brianlittmann** [summernote/summernote-rails#38]
Good work on the lib, guys. Everything seems to be working well, except when loading or initializing a form that has HTML content already in it. For example, I create some markup in the form:

![before saving](https://cloud.githubusercontent.com/assets/1688640/12026273/52106134-ad7d-11e5-892b-8f5624252f24.png)

Save, gets written to the database correctly, but when I reload the form to edit, the HTML is simply raw when I would expect it to be formatted as before:

![after saving](https://cloud.githubusercontent.com/assets/1688640/12026283/6f8a5b52-ad7d-11e5-8ac9-b8e67e717df1.png)

Thoughts?

**Comment by brianlittmann** [comment]
Figured it out. Looks like v0.7 no longer uses a `textarea` input, but a `div` instead. Below is what I got working, by manually setting the content of a hidden `textarea` form input. It would probably be a good idea to update the README docs though.

```erb
<div class="form-group">
	<%= f.label :description %>
	<%= f.text_area :description, class: "form-control", rows: 10, id: "content-description" %>
	<div id="summernote-description" style="display: none;">
		<%= f.object.description.html_safe %>
	</div>
</div>
```

```javascript
var ContentDescription = {
	htmlInput: $('#summernote-description'),
	formInput: $('#content-description'),

	init: function () {
		this.formInput.hide();
		this.htmlInput.show();

		this.htmlInput.summernote({
			callbacks: {
				onChange: function (e) {
					ContentDescription.formInput.html(ContentDescription.htmlInput.summernote('code'));
				}
			},
			toolbar: [
				// http://summernote.org/deep-dive/#custom-toolbar
				['style', ['bold', 'italic', 'underline']],
				['list', ['ul', 'ol']],
				['link', ['link']]
			],
			disableDragAndDrop: true,
			height: 200
		});
	}
}
```

**Comment by luciuschoi** [comment]
Good! <3

**Comment by seapy** [comment]
it's summernote 0.7.0 bug. fixed 0.7.1 and released soon.
see this issue https://github.com/summernote/summernote/issues/1468
